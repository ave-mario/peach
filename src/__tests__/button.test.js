import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from 'shared/buttons.css';
import 'jest-styled-components';

describe('Buttons:', () => {
  it('component rednders correctly', () => {
    const ButtonComponent = renderer.create(<Button />).toJSON();
    expect(ButtonComponent).toMatchSnapshot();
  });

  it('check cancel button styles', () => {
    const ButtonComponent = renderer.create(<Button cancel />).toJSON();
    expect(ButtonComponent).toHaveStyleRule('color', 'red');
  });
});
