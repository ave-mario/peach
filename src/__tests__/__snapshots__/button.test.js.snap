// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons: component rednders correctly 1`] = `
.c0 {
  border: none;
  cursor: pointer;
  font-weight: bold;
  outline: none;
  padding: 10px;
}

.c0:disabled {
  opacity: 0.4;
}

<button
  className="c0"
/>
`;
